packages/cli/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@solana-mobile/dapp-store-cli",
-  "version": "0.8.2",
```

Comment:
What mandatory changes are we pushing in this release?

Reply:
1. Icon enforcement  (icon size 512x512)
2. Video support  (new feature)
3. Screenshot enforcement (at least 4 screenshots or videos)
4. Abi checks (non mandatory warning)